### ISSUE TITLE
Current code isn't supported by Discord anymore.

### ISSUE THREAD

**Issue by Olkris** [Firstbober/Krita-rpc#2]
I'm not too advanced but it says on the pypresence patch notes that people should stop using things like "activities" because they're deprecated or something.

**Comment by Firstbober** [comment]
As far as I know, the entire Rich Presence stack is deprecated in favor of Game SDK and its Activity Manager! I'm curious if it's possible to port this to Game SDK.

**Comment by Olkris** [comment]
I think their Game SDK is mainly a name rebrand, with a few things removed, and some extra features like game party invites, etc.

It's not just "rich presence" anymore, some parts of it have to be put in the game for the extra features, so it's an SDK.

I forked your program and with my limited knowledge and a few backspaces I managed to get it working again. So that would indicate that indeed it's mostly stayed the same. Tho I didn't read the docs so idk.
